DESCRIPTION

```diff
@@ -1,6 +1,5 @@
 Package: EpiEstim
-Version: 2.0-0
-Date: 2012/01/17
```

Comment:
Is there a reason this has been removed? Is this is standard thing to have that should be updated?

Reply:
Date is not really used anymore, so it's fine (and preferred) to leave it out.